package.json

```diff
@@ -29,6 +29,7 @@
     "pino": "^7.6.4",
     "pino-pretty": "^7.5.0",
     "sinon": "^12.0.1",
+    "swagger": "^0.7.5",
```

Comment:
i think we are not going to this package specifically but instead we are going to use this one swagger-ui-express https://www.npmjs.com/package/swagger-ui-express so we can serve all the api from json file